### PR TITLE
Initial Tutorials for lib functionality before rc1

### PR DESCRIPTION
**Pull request by kontogiannisg**

### CODE REVIEW

**Comment by CaptainAlready** [comment]
The tutorials contain many errors which are also printed and increase the file size.
Could some prints also be reduced to decrease the size of the tutorial notebooks?
Also, in my opinion the tutorial notebooks are too many (1 - 2  small tutorials would suffice)

**Comment by tzamalisp** [comment]
> The tutorials contain many errors which are also printed and increase the file size. Could some prints also be reduced to decrease the size of the tutorial notebooks? Also, in my opinion the tutorial notebooks are too many (1 - 2 small tutorials would suffice)

@CaptainAlready we are under development progress here. Therefore, we will take into account all the possible ways to reduce the tutorials' file size. According to the number of tutorials, we are designing with @gkontogiannhs which will be the best way for the user to understand how the lib works.

**Comment by kontogiannisg** [comment]
Thank you, @CaptainAlready, for your feedback. The Tutorials branch aims to assess the robustness of the library in its initial phase. Through this, we'll explore potential feature redesigns to enhance user-friendliness, followed by optimizations for size, clarity, and overall efficiency. Regarding errors, some were intentionally left unaddressed due to lack of data in certain cases. Rest assured, these issues will be promptly resolved.

**Comment by tzamalisp** [comment]
@gkontogiannhs 

## Configuration

in the following lines https://github.com/tzamalisp/crossai/blob/57c661e6e072e585d63b6b020b162dcb27d4a0b2/tutorials/config_loader.py#L4-L13

* add properly the "Loads configuration from.." next to the `"""`

## Audio Speech Commands Recognition

At the start of the tutorial, I would add some extra information about the dataset. The NB looks like the one that I created a few months ago.

in the lines

```
os.environ["CUDA_VISIBLE_DEVICES"] = "2"
os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
```
add the level to the configuration file.

in `DATASET_PATH = config['PATH']['MINI_SPEECH_COMMANDS']`, please define how the user should load the data properly (audio data in classes, link to the README)

why this one is in comment: `# tf.keras.utils.plot_model(cnn1d, to_file='Simple-CNN1D.png', show_shapes=True)`

the next one could go to a `utils` script:

```
def plot_learning(history):
    metrics = history.history
    history_keys = list(history.history.keys())
    plt.figure(figsize=(16, 6))
    plt.subplot(1, 2, 1)
    plt.plot(history.epoch, metrics[history_keys[0]], metrics[history_keys[2]])
    plt.legend(['loss', 'val_loss'])
    plt.ylim([0, max(plt.ylim())])
    plt.xlabel('Epoch')
    plt.ylabel('Loss [CrossEntropy]')

    plt.subplot(1,2,2)
    plt.plot(history.epoch, 100*np.array(metrics[history_keys[1]]), 100*np.array(metrics[history_keys[3]]))
    plt.legend(['accuracy', 'val_accuracy'])
    plt.ylim([0, 100])
    plt.xlabel('Epoch')
    plt.ylabel('Accuracy [%]')
``` 
and add more epochs for better visualization

**Comment by tzamalisp** [comment]
@gkontogiannhs @fothot2 

## Motion Recognition E2E

* In cell 11, I would add only once the `TimeMeasurementWrapper`, to show the functionality.  I would also explain what it does in a few words in the markdown.

* the `plot_ts` could export two subplots so the y-axis is depicted properly in case of a huge difference between the range of values of the different time series.

* the `batch_evaluate` should count and indicate the instance that it processes each time.

File: tutorials/config_loader.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import yaml
+
+
+def load_config(config_file_path: str):
+    """
```
Comment (tzamalisp) [inline tutorials/config_loader.py]: add properly the "Loads configuration from.." next to the """

**Review by tzamalisp — CHANGES_REQUESTED** [review]
